packages/reducer/README.md

```diff
@@ -0,0 +1,80 @@
+# @solid-primitives/reducer
+
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg?style=for-the-badge)](https://lerna.js.org/)
+[![size](https://img.shields.io/bundlephobia/minzip/@solid-primitives/reducer?style=for-the-badge&label=size)](https://bundlephobia.com/package/@solid-primitives/reducer)
+[![version](https://img.shields.io/npm/v/@solid-primitives/reducer?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/reducer)
+[![stage](https://img.shields.io/endpoint?style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fsolidjs-community%2Fsolid-primitives%2Fmain%2Fassets%2Fbadges%2Fstage-0.json)](https://github.com/solidjs-community/solid-primitives#contribution-process)
+
+SolidJS equavalent of React [useReducer](https://reactjs.org/docs/hooks-reference.html#usereducer).
+
+## Installation
+
+```bash
+npm install @solid-primitives/reducer
+# or
+yarn add @solid-primitives/reducer
+```
+
+## How to use it
+
+```ts
+const [acessor, dispatch] = createReducer(reducer, initialValue);
```

Comment:
Please add the type definitions to show users not completely familiar with react and/or the flux pattern what is to be expected here, i.e.

```ts
const [accessor, dispatch] = createReducer<State, Action>(
  reducer: (state: State, action?: Action) => State,
  initialValue: State
);
accessor(); // State
dispatch({ type: 'increment' });
```

Reply:
A short text description of what it is and when you'd want to use it would be valuable too.
Because as I understand this isn't supposed to be a tool only for react developers to use.